src/applications/representative-appoint/components/ClaimantType.jsx

```diff
@@ -4,10 +4,12 @@ import { connect, useSelector } from 'react-redux';
 import { getNextPagePath } from '~/platform/forms-system/src/js/routing';
 import { setData } from '~/platform/forms-system/src/js/actions';
 import { toggleLoginModal as toggleLoginModalAction } from '~/platform/site-wide/user-nav/actions';
+import { isLoggedIn } from 'platform/user/selectors';
 import ClaimantTypeForm from './ClaimantTypeForm';
+import prefillTransformer from '../config/prefillTransformer';
 
 const ClaimantType = props => {
-  const { router, setFormData } = props;
+  const { router, setFormData, loggedIn } = props;
 
   const { data: formData } = useSelector(state => state.form);
```

Comment:
This isn't one of your changes, but I think `formData` should also be retrievable from props and `useSelector` shouldnt be necessary? If it's an easy update then we may as well make it now, otherwise I can approve.

Reply:
I switched to getting the form data from state like our other components